README.md

```diff
@@ -72,6 +72,10 @@ verbose = true
 ### Hooks
 Under the `[hooks]` table, you can add an entry for any and every git hook you want to run by adding a key using the name of the [git hook][git hooks], and then specify the command/script you want to run for that hook. Whenever that git hook is triggered, `rusty-hook` will run your specified command!
 
+#### Using git arguments
+In git hook commands, any instance of `%@` will be replaced by the arguments that git passes to this hook.  
+**Be warned, this CAN be dangerous if used incorrectly!**
```

Comment:
I'm thinking to err on the safe side we may want a longer replacement token sequence. It definitely seems the possibility of collisions with valid/non-desired replacement is low, but there's no harm in having something longer than two chars.

Also, could you elaborate a bit on the warning? `dangerous` in the sense of a hook not doing what a user would expect/failing due to an extra arg or in the sense of a security vulnerability, etc.

Would probably be good to include an example as well if you wouldn't mind adding one!

Reply:
Dangerous as in "command injection if you have untrusted git args... somehow"

Reply:
I'd have to refresh my memory but to the best of my recollection the actual arg values aren't open-ended. They'll be things like a local file (for example the proposed commit message log file), remote/branch/etc. names, refs/SHA's, a variant from git defined enums that indicate which git flag/option was provided, etc.

Reply:
Aight!

Reply:
Thanks for updating with an example! I would still like to change the replacement sequence to something longer than two characters (even something like `%@%@`). Do you have any concerns or strong objections to making this value something longer?

I also want to remove the text with the dangerous behavior warning unless there's any specific instances that can be identified. Reviewing the [git hook docs](https://git-scm.com/docs/githooks#_hooks) confirms what I said above regarding the fixed nature of possible values.

I'm probably going to go ahead and implement #25 after merging this and then do a release with these recent new features, so if you'd rather leave the text as-is for now then I can just update it as part of that.